people/jakub.json

```
@@ -0,0 +1,12 @@
+{
+  "key": "jakub",
+  "name": "MUDr. Jakub Švec",
+  "nick": "Jakub",
+  "email": "jakub.svec@mild.blue",
+  "title_cs": "ODBORNÝ KONZULTANT",
```

```suggestion
  "title_cs": "Odborný konzultant",
```

Píšme prosím sem do .json tie pozície bez kapitáliek (viď .json ostatných), teraz to na webe máme kapitálkami, ale budeme to možno chcieť zmeniť a potom by si to mal inak, ako ostatní.  to isté pre `en` a `de` prosím, ďakujem.

---

Opravím. Jen nedělá Jakub (nemá Git), ale Jana U.. :)

---

Opravené v komentári 😉